ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host3/device-documentation-toc.yml

```diff
@@ -0,0 +1,2 @@
+---
+device_documentation_toc: false
```

Comment:
I think this would fit better here
``yaml
eos_cli_config_gen_documentation:
  <...>
  toc: <bool; default=true>
```

Reply:
updated